Dockerfile

```diff
@@ -22,10 +22,9 @@ WORKDIR /go/src/github.com/cosmos/cosmos-sdk
 # Add source files
 COPY . .
 
-# build Cosmos SDK, remove packages
-RUN make build-simd && \
-    cp ./build/simd /go/bin
-# make build-sim-linux ??
+# install simapp, remove packages
+RUN make install
```

Comment:
This does not build simd. Why not calling `make simd GOOS=linux GOARCH=amd64`?

Reply:
install or build they end up doing the same thing in the end because I would copy ./build/simd into bin.